docs/INSTALL.rst

```diff
@@ -60,3 +60,13 @@ Or from GitHub::
 
 where the dependencies for orchestrating are installed. They are used for
 interacting with the services to schedule tests and to report the test results.
+
+
+Update Dependencies
+-------------------
+
+We track the dependencies using ``requirements.txt``. These packages are tested,
+and should work with teuthology. But if you want to bump up the version of them,
+please use the following command to update this file ::
+
+  echo "-e .[coverage,orchestra,test]" | pip-compile - -qo- | sed '/^-e / d' > requirements.txt
```

Comment:
What about changing this to:
```
pip-compile -qo- | sed '/^-e / d' > requirements.txt
```
and commit `requirements.in`:
```
-e .[coverage,orchestra,test]
```
Also we can have `update_requirements` shell script in the root directory?

Reply:
i think the document is good enough. if we really believe the usability the one-liner shell script is a pain, i will fix it in a follow-up PR.

Reply:
I'm fine with the oneliner, trying to find better to way to manage `coverage,orchestra,test` dependencies. It is more intuitive to look for them in requirements.in instead of modifying the doc each time new dependency is added or removed.

Reply:
Usually, updating the docs is the last thing a developer thinks.

Reply:
normally, it's not necessary to update the document for adding or removing dependencies. in this change, it's needed only if a new subset of extra requirement is added.

Reply:
normally, it's not necessary to update the document for adding or removing dependencies. in this change, it's needed only if a new *subset* of extra requirement is added.

Reply:
as you insist, i added `requirements.in` in the latest change.  but please note, it's only a workaround and a hack. and it's not quite right. because the `requirements.in` is pointing to the package itself. i'd avoid adding such a file if i could.

also, i don't want to add the helper script. and i explained the reason in https://github.com/ceph/teuthology/pull/1367#discussion_r353896681 . as the document already put the way how `requirements.txt` is updated. another script just adds another layer of abstraction which is overkill.